### DISCUSSION TITLE
Playtime 2 occasional display issue with UI being completely white on launch.

### DISCUSSION THREAD

**Discussion by FakenameMusic** [helgoboss/helgobox#1169] [General]
Every so often when I launch Playtime 2 it launches with a completely white screen. Functionality works, but I cannot see the UI at all. I can click "X" to close the window because I know where it is, and relaunch it and it usually works the second time. (I'll add a screenshot here the next time it happens - but it's literally just a white screen - no text or UI elements visible) Let me know if any other info would be of use.

**Comment by helgoboss** [comment]
https://github.com/helgoboss/helgobox/issues/1159

**Reply by FakenameMusic** [reply]
Thank you. My search did not find this. I will post any further details in that thread.
